README.md

```diff
@@ -1,568 +1,181 @@
+        "top_p": 1,
+        "temperature": 0,
+        "system_prompt": "You are an expert travel coordinator with exquisite taste. Be concise but specific in recommendations.\n\nEmoji: Choose an emoji based on the location: \n\nOutput Format: \n## Personalized  Itinerary [emoji]\n&nbsp;\n### Morning\n[Breakfast spot and attraction]\n&nbsp;\n### Afternoon\n[Lunch spot and attraction]\n&nbsp;\n###Evening\n[Dinner spot and attraction]\n&nbsp;\n### Night\n[Dessert spot and attraction]\n\nStyle Guidelines: \nBold the restaurants and the attractions. Be structured."
       }
     },
-    "default_model": "gpt-3.5-turbo"
+    "default_model": "gpt-3.5-turbo",
+    "parameters": {"city": "London"}
   },
   "prompts": [
     {
-      "name": "get_activities",
-      "input": "Tell me 10 fun attractions to do in NYC."
+      "name": "get_activities", 
+      "input": "Give me the top 5 fun attractions to do in {{city}}"
     },
     {
       "name": "gen_itinerary",
-      "input": "Generate an itinerary ordered by {{order_by}} for these activities: {{get_activities.output}}.",
+      "input": "Generate a one-day personalized itinerary based on : \n1. my favorite cuisine: {{cuisine}} \n2. list of activities:  {{get_activities.output}}",
       "metadata": {
-        "model": "gpt-4",
-        "parameters": {
-          "order_by": "geographic location"
-        }
+        "model": {"name": "gpt-4"},
+        "parameters": {"cuisine": "Malaysian"},
+        "remember_chat_context": false
       }
     }
   ]
 }
-```
-
-### Run the `get_activities` prompt.
-
-You don't need to worry about how to run inference for the model; it's all handled by AIConfig. The prompt runs with gpt-3.5-turbo since that is the `default_model` for this AIConfig.
-
-Create a new file called `app.py` and and enter the following code:
-
-```python
-import asyncio
-from aiconfig import AIConfigRuntime, InferenceOptions
+</code></pre>
+</details>
 
-async def main():
-  # Load the aiconfig
-  config = AIConfigRuntime.load('travel.aiconfig.json')
+<br>
 
-  # Run a single prompt (with streaming)
-  inference_options = InferenceOptions(stream=True)
-  await config.run("get_activities", options=inference_options)
-
-asyncio.run(main())
-```
-
-Now run this in your terminal with the command:
-
-```bash
-python3 app.py
-```
-
-### Run the `gen_itinerary` prompt.
-
-In your `app.py` file, change the last line to below:
-
-```python
-await config.run("gen_itinerary", params=None, options=inference_options)
-```
-
-Re-run the command in your terminal:
+The core SDK allows you to use your AIConfig easily in your application code.
+Code below uses `trip_planner_aiconfig.json` shared above.
+Resources: [Getting Started Docs](https://aiconfig.lastmileai.dev/docs/getting-started) | [Google Colab](https://colab.research.google.com/drive/1RlGQmtR0uK7OTI5nG10E219JoH2mgAQr#scrollTo=h2G7ThhyFWxg) | [Demo Video](https://www.youtube.com/watch?v=X_Z-M2ZcpjA)
```

Comment:
Add Demo video as an embed:
https://www.youtube.com/watch?v=X_Z-M2ZcpjA&t=2s

i.e. `[![Watch the video](https://img.youtube.com/vi/X_Z-M2ZcpjA/default.jpg)](https://www.youtube.com/watch?v=X_Z-M2ZcpjA)`